travis/.rubocop_rspec_base.yml

```diff
@@ -114,10 +114,61 @@ SpaceAroundEqualsInParameterDefault:
 StringLiterals:
   Enabled: false
 
+# By convention, RSpec examples use expect { ... } rather than expect do ... end
+Style/Blocks:
+  Exclude:
+    - spec/**/*
+
+# In tests, it is convenient to define a test class like RSpec::Something on a single line
+Style/ClassAndModuleChildren:
+  Exclude:
+    - spec/**/*
+
+# Our tests use compact class definitions to save "real estate" on the scren.
+Style/EmptyLineBetweenDefs:
+  Exclude:
+    - spec/**/*
+Style/EmptyLinesAroundAccessModifier:
```

Comment:
I deliberately put them close together, since they are conceptually related. But I can separate them if you like.